README.md

```diff
@@ -8,3 +8,4 @@
 | № | Имя Фамилия     | Любимый фильм                          |
 | - |:---------------:| --------------------------------------:|
 | 0 | Ава Катушка     | Три идиота                             |
+| 4 | Веня Стаховский | Интерстеллар                           |
```

Comment:
How is it possible that after number 0 follows number 4?

Reply:
*number 4 goes after number 0. It shouldn't be. Please fix.

Reply:
I used number №4 because there were 3 pull requests before me

Reply:
Well, it's not how git works. You should ignore all the other pull requests until they are accepted and merged. 
But you should pull changes that did already happen to your repository.

Reply:
You should also resolve the conflicts.

Reply:
Ava, Sorry, i don't understand how to pull changes from the main repository to my own repository.

Reply:
You can start with resolving conflicts: 
![2021-02-22_17-57-47](https://user-images.githubusercontent.com/73282400/108725984-fb051f00-7537-11eb-990c-f35d740a84f7.png)